h3_test.go

```diff
@@ -283,34 +407,75 @@ func TestDirectedEdge(t *testing.T) {
 
 	t.Run("get origin/destination from edge", func(t *testing.T) {
 		t.Parallel()
-		assertEqual(t, origin, edge.Origin())
-		assertEqual(t, destination, edge.Destination())
+		originExpected, err := edge.Origin()
+		assertNoErr(t, err)
+		assertEqual(t, origin, originExpected)
+
+		destinationExpected, err := edge.Destination()
+		assertNoErr(t, err)
+		assertEqual(t, destination, destinationExpected)
 
 		// shadow origin/destination
-		cells := edge.Cells()
+		cells, err := edge.Cells()
+		assertNoErr(t, err)
+
 		origin, destination := cells[0], cells[1]
-		assertEqual(t, origin, edge.Origin())
-		assertEqual(t, destination, edge.Destination())
+		originExpected, err = edge.Origin()
+		assertNoErr(t, err)
+		assertEqual(t, origin, originExpected)
+
+		destinationExpected, err = edge.Destination()
+		assertNoErr(t, err)
+		assertEqual(t, destination, destinationExpected)
 	})
 
 	t.Run("get edges from hexagon", func(t *testing.T) {
 		t.Parallel()
-		edges := validCell.DirectedEdges()
+		edges, err := validCell.DirectedEdges()
+		assertNoErr(t, err)
 		assertEqual(t, 6, len(edges), "hexagon has 6 edges")
 	})
 
 	t.Run("get edges from pentagon", func(t *testing.T) {
 		t.Parallel()
-		edges := pentagonCell.DirectedEdges()
+		edges, err := pentagonCell.DirectedEdges()
+		assertNoErr(t, err)
 		assertEqual(t, 5, len(edges), "pentagon has 5 edges")
 	})
 
 	t.Run("get boundary from edge", func(t *testing.T) {
 		t.Parallel()
-		gb := edge.Boundary()
+		gb, err := edge.Boundary()
+		assertNoErr(t, err)
 		assertEqual(t, 2, len(gb), "edge has 2 boundary cells")
 	})
 }
+func testDirectedEdgeError(t *testing.T) {
+	t.Parallel()
+	_, err := Cell(0x891ea6d6533ffff).DirectedEdges()
+	assertTrue(t, errors.Is(err, H3ErrorInvalidDirEdge))
+
+	_, err = validCell.DirectedEdge(validDiskDist3_1[2][0])
+	assertTrue(t, errors.Is(err, H3ErrorNeighbors))
+
+	_, err = DirectedEdge(0).Origin()
+	assertTrue(t, errors.Is(err, H3ErrorInvalidDirEdge))
+	_, err = DirectedEdge(0x115283773fffffff).Origin()
+	assertTrue(t, errors.Is(err, H3ErrorFailed))
+
+	_, err = DirectedEdge(0).Destination()
+	assertTrue(t, errors.Is(err, H3ErrorInvalidDirEdge))
+	_, err = DirectedEdge(0x115283773fffffff).Destination()
+	assertTrue(t, errors.Is(err, H3ErrorFailed))
+
+	_, err = DirectedEdge(0).Cells()
+	assertTrue(t, errors.Is(err, H3ErrorInvalidDirEdge))
+	_, err = DirectedEdge(0x115283773fffffff).Cells()
+	assertTrue(t, errors.Is(err, H3ErrorFailed))
+
+	_, err = DirectedEdge(0x115283773fffffff).Boundary()
+	assertTrue(t, errors.Is(err, H3ErrorInvalidDirEdge))
```

Comment:
can you pop the literal addresses into a named fixture var at the top of the file, similar to the other fixture vars? it helps us keep a handle on what these values mean.